tests/cli/test_account.py

```diff
@@ -0,0 +1 @@
+from anchorecli.cli import account
```

Comment:
in all these tests files there aren't any tests, but just having them imported like this will allow us to have some structure *and* at least know if the module imports even though the file is mostly empty